nfadeeva/assignment2/Assignment_2_Q2.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.right = None
+        self.left = None
+
+
+def find_lca(root, node1, node2):
+    """
+        Find the lowest common ancestor of two nodes in a binary tree
+        :type node1: Node
+        :type node2: Node
+        :rtype: Node or False
+        :return The lowest common ancestor of two nodes in a binary tree
```

Comment:
Could you raise a KeyError if one of the Node is not in the given tree or both of them are not in the tree?